classes/plugins/PluginHelper.inc.php

```diff
@@ -240,7 +244,11 @@ function upgradePlugin($category, $plugin, $path, &$errorMsg) {
 
 			$upgradeFile = $pluginDest . '/' . PLUGIN_UPGRADE_FILE;
 			if($fileManager->fileExists($upgradeFile)) {
+                                $request = Application::getRequest();
+                                $site = $request->getSite();
```

Comment:
Rather than using the `Request` object, which won't be available when invoked from the command line, I'd suggest using

    $siteDao = DAORegistry::getDAO('SiteDAO').
    $site = $siteDao->getSite();